headeremitter.js

```diff
@@ -72,10 +72,17 @@ function clamp(value, min, max, def) {
  *   @param [options.useASCII=true] {Boolean}
  *     If true, then RFC 2047 and RFC 2231 encoding of headers will be performed
  *     as needed to retain headers as ASCII.
+ *   @param [options.useOnlyUTC=true] {Boolean}
+ *     If true, then date added in header is always in UTC. This is to provide 
+ *     user location privacy by not revealing Timezone location.
```

Comment:
The option is probably better named "hideDateTimezone". Also, your English is not grammatically correct here:

If true, then use the UTC time zone instead of the local time zone when emitting date values.

[No need to mention the second sentence]